lib/integrations/hip_chat.rb

```diff
@@ -51,5 +51,13 @@ def message_color
 
       color_hash[event_name]
     end
+
+    def run_href
+      "<a href=\"#{payload[:frontend_url]}\">Run ##{run[:id]}#{run_description}</a>"
```

Comment:
Is there a reason not to use this as the default? I would have assumed most integrations would support HTML? Just a thought.